test-integration/helpers/ui/test-element.coffee

```diff
@@ -62,71 +88,68 @@ class TestItemHelper
       else
         false
 
-  # Helper for the common case of `get(...).click()`.
-  # Plus, it allows a place to add logging since this is one of the most
-  # common places for Selenium to time out (trying to click on an element)
-  click: (args...) =>
-    locator = @getLocator(args...)
-    # Scroll to the element so it is visible before clicking (this assumes `position: fixed` is overridden for all element)
-    # @isDisplayed(args...).then (isDisplayed) =>
-    #   unless isDisplayed
-    #     el = @findElement(args...)
-    #     @test.utils.windowPosition.scrollTo(el)
-    @test.utils.verboseWrap "Clicking #{JSON.stringify(locator)}", =>
-      el = @get(args...)
-      @test.utils.windowPosition.scrollTo(el)
-      el.click()
-
-  waitClick: (args...) =>
+  getParent: (args...) =>
     locator = @getLocator(args...)
-    @test.utils.verbose "Waiting for #{JSON.stringify(locator)}"
-    @test.utils.wait.for(locator)
-    @click(args...)
+    @test.utils.dom.getParent(locator)
 
 
-class TestHelper extends TestItemHelper
+class TestHelper
   constructor: (test, testElementLocator, commonElements, options) ->
-    super(test, testElementLocator)
     commonElements ||= _.result(@, 'elementRefs', {})
+    commonElements = _.extend({}, DEFAULT_ELEMENTS, commonElements)
```

Comment:
@philschatz how about this?  this makes it so that `loadingState` in on the `commonElements` argument, has a default value of `css: .is-loading`, and can be overridden with custom `loadingState` if needed as part of the `commonElements` argument